src/fu-self-test.c

```diff
@@ -6198,6 +6199,27 @@ fu_remote_local_func(void)
 	g_assert_true(ret);
 }
 
+static void
+fu_remote_list_repair_func(void)
```

Comment:
The tests should cover both the positive and negative cases.

Reply:
Done.